rtnetlink/src/traffic_control/get.rs

```diff
@@ -126,3 +126,42 @@ impl TrafficFilterGetRequest {
         }
     }
 }
+
+pub struct TrafficChainGetRequest {
+    handle: Handle,
+    message: TcMessage,
+}
+
+impl TrafficChainGetRequest {
```

Comment:
As in #91 a small explanation and equivalent `tc` commands would be nice.

Reply:
The document is at the `rtnetlink/src/handle.rs` like others PRs:

```rust
    /// Create a new handle, specifically for traffic control chain requests
    /// (equivalent to `tc chain show dev <interface_name>` commands)
    pub fn traffic_chain(&self, ifindex: i32) -> TrafficChainHandle {
        TrafficChainHandle::new(self.clone(), ifindex)
    }
```